server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -27,12 +27,6 @@ public class FeatureFlags {
     public static final String SEGMENT_REPLICATION_EXPERIMENTAL =
         "opensearch.experimental.feature.segment_replication_experimental.enabled";
```

Comment:
we would need to remove this as well as remote store needs to be integrated segrep.

Reply:
Lets remove this. If we need to add something in future, we can add back the feature flag or a dynamic setting?

Reply:
Lets do this later in view of code freeze .